playbooks/roles/nomad/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+nomad_version: "0.8.3"
+nomad_url: "https://releases.hashicorp.com/nomad/{{ nomad_version }}/nomad_{{ nomad_version }}_linux_amd64.zip"
+
+nomad_server: false
+nomad_client: false
+nomad_enable_tls: true
+nomad_enable_telemetry: true
+
+nomad_bin_dir: /usr/local/bin
+nomad_data_dir: /var/lib/nomad
+nomad_config_dir: /etc/nomad
+nomad_tls_dir: "{{ nomad_config_dir }}/tls"
+
+nomad_client_reserved_cpu: 500 # MHz
```

Comment:
This seems quite a bit – is this the recommened amount, or did you just make this up?  (Most schedulers I know use fractions of CPUs as a unit, which I find easier to handle, but I can see why Nomad chose to use MHz instead.)

Reply:
I took this value straight from the document examples for lack of a better value, but it definitely did seem like too much to me too, and I planned on changing it. How much do you think I should put here though, that'll be required by things like Filebeat, Consul, the kernel, etc?

Reply:
Thinking about it, I think we should just not reserve CPU at all, but reserve only memory instead. We can probably make do with other services, at least Filebeat & the Consul client agent, running a bit slower because of lack of CPU cycles being allocated to them, since I think only memory shortage would cause visible issues.

Reply:
Sounds good to me – I also expect memory to be the limiting resource, not CPU.  We can always change it after we gain some experience.